service-registry/src/main/java/org/apache/servicecomb/serviceregistry/task/MicroserviceInstanceRegisterTask.java

```diff
@@ -68,6 +68,10 @@ protected boolean doRegister() {
     microserviceInstance.getHealthCheck().setTimes(serviceRegistryConfig.getResendHeartBeatTimes());
 
     String instanceId = srClient.registerMicroserviceInstance(microserviceInstance);
+    MicroserviceInstance temIns = srClient.findServiceInstance(microserviceInstance.getServiceId(),instanceId);
+    if(temIns!=null){
```

Comment:
Use the code formatter in etc folder to format code

Reply:
done